noir-projects/noir-contracts/contracts/test_contract/src/test.nr

```diff
@@ -77,3 +82,89 @@ fn get_shared_mutable_num_slots() -> Field {
 
     value_change_and_delay_change_packed_len + with_hash_additional_slots
 }
+
+pub unconstrained fn setup() -> (&mut TestEnvironment, AztecAddress, AztecAddress) {
+    // Setup env, generate keys
+    let mut env = TestEnvironment::new();
+    let owner = env.create_account(1);
+    env.impersonate(owner);
+
+    // TODO: (12276) This is here because we are testing historical proofs, and we cannot get a block_header at 1 due to a bug in world state.
```

Comment:
```suggestion
    // TODO(#12276): This is here because we are testing historical proofs, and we cannot get a block_header at 1 due to a bug in world state.
```

Just a nit but I find this a better format as "#12276" is more likely to be unique in the codebase.